Deploy/Systemd/ss-media-daemon.service.app

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Media Daemon
```

Comment:
Why the name is still so vague? If we look into system log, how can we know what service is this? Is it of the OS or ours?